src/tunnel.ts

```diff
@@ -1,7 +1,7 @@
 import net from 'net';
 import { Client } from 'ssh2';
 import type { ConnectConfig } from 'ssh2';
-import { getPort, readFile } from './utils';
```

Comment:
No need anymore, but I have not removed the portfinder dependency yet. I will leave that to another PR.

Reply:
I think it's fine to remove within this PR given there's no other PRs open right now touching lock files or whatever and it's no longer used anymore.